diff --git a/run.sh b/run.sh
index 83f5bcd..217a39b 100755
--- a/run.sh
+++ b/run.sh
@@ -1,4 +1,4 @@
 #env FLASK_APP=BigBroCast.py 
 #python3 -m flask run
 
-python3 -m BigBroCast
+python3.7 -m BigBroCast
diff --git a/shellscripts/set_vol.sh b/shellscripts/set_vol.sh
index d7f9575..efeb89f 100755
--- a/shellscripts/set_vol.sh
+++ b/shellscripts/set_vol.sh
@@ -1 +1 @@
-amixer -D pulse sset Master $1%
+amixer sset 'Line Out' $1%

